GiniVision/Classes/CameraViewController.swift

```diff
@@ -303,60 +316,86 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         }
         self.previewView.addSubview(imageView)
         UIView.animate(withDuration: 1.5,
-                                   animations: {
-                                    imageView.alpha = 0.0
-            },
-                                   completion: { (success: Bool) -> Void in
-                                    imageView.removeFromSuperview()
+                       animations: {
+                        imageView.alpha = 0.0
+        },
+                       completion: { (success: Bool) -> Void in
+                        imageView.removeFromSuperview()
         })
     }
     
     // MARK: Constraints
     fileprivate func addConstraints() {
-        let superview = self.view
-        
-        // Preview view
+        addPreviewViewConstraints()
+        addCameraOverlayConstraints()
+        addControlsViewConstraints()
+        addCaptureButtonConstraints()
+    }
+    
+    fileprivate func addPreviewViewConstraints(){
```

Comment:
Usually we put a space between the ")" and "{". I'd prefer it if you did the same to keep it consistent.